### PR TITLE
Update User Guide for Revenue 

### LINKED ISSUE #63: Update User Guide for Revenue

### PR DESCRIPTION
**Pull request by galvinleow**
Closes #63

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -131,29 +127,127 @@ The table below shows a list of command parameters that will be used in this fea
...
 Outcome : 
-1. It will display a success message ~~in the *Dashboard*.~~
-1. GrAB3 will list out all the revenue for 29-09-2020 ~~in the *Revenue* tab.~~  
-1. You will be able to see all the revenue for 29-09-2020.
+1. It will display a success message ~~in the _Dashboard_.~~
+1. GrAB3 will list out all the revenue for 29-09-2020 in the _Revenue_ tab.  
+
+{Example outcome screenshot}
+
+##### Sort Revenues: `sortrev`
```
Comment (khoongwk) [inline docs/UserGuide.md:159]: See #62 from Hans, he included a numbering scheme for the features like 4.1.1, 4.1.2, etc. I think you can edit this section after his PR is merged into master.

**Review by khoongwk — CHANGES_REQUESTED** [review]
I think it's a good idea to include a numbering system like what @hansebastian did in #62.

**Review by yanlynnnnn — COMMENTED** [review]
LGTM 👍🏻

File: docs/UserGuide.md (hunk 1 above)
Comment (galvinleow) [reply docs/UserGuide.md:159]: I will do this in the last milestone, issue created

**Review by khoongwk — APPROVED** [review]

**Comment by khoongwk** [comment]
Numbering scheme will be relegated to v1.4

**Pull request merged**